### ISSUE TITLE
Blocked aria-hidden on covers from the home page

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#2718]
![image](https://github.com/user-attachments/assets/a92151c5-4ed2-4051-ba71-e7ea439ae9d4)

From AXE A11Y checker

> index_library.html#/home:1 Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
> Element with focus: img
> Ancestor with aria-hidden:  <img tabindex=​"-1" class=​"cover_img" role=​"presentation" alt aria-hidden=​"true" src=​"store:​/​/​39b7940f-df1a-4e64-a8a9-99d872c2985f/​cover.png">​flex

redundant on each cover image

I guess deleting `aria-hidden=​"true"` should be a sufficient fix because `role=​"presentation"` already does the job
